### PR TITLE
:memo: update deck link

### PR DESCRIPTION
**Pull request by fabienjuif**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Welcome to our source speech repository about [flubber](https://github.com/veltm
  - Fabien JUIF [[Twitter](https://twitter.com/fabienjuif) | [Github](https://github.com/fabienjuif)]
 
 ## Deck
- - Soon
+ - [BDX.io Bordeaux, France - 2017 /en/](https://docs.google.com/presentation/d/16uJIRTJbvknvPyrH_nE-AmLdVBKw8hy6Rk6aAggrpYk/edit?usp=sharing)
```
Comment (delphinemillet) [inline README.md]: For PMP repo, we used the following format : [French version](...)
Do we need to share each version of the deck (for each time we played this talk :p) ?
In my opinion, the last version is suffisent

Comment (fabienjuif) [reply README.md]: I prefers to have one version per event in case we change the deck.
We can add a "latest version" if you want

Comment (delphinemillet) [reply README.md]: If someone visits the repo, he may be confused on don't know which version he should consult, don't you think ?

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Welcome to our source speech repository about [flubber](https://github.com/veltm
  - Fabien JUIF [[Twitter](https://twitter.com/fabienjuif) | [Github](https://github.com/fabienjuif)]
 
 ## Deck
- - [BDX.io Bordeaux, France - 2017 /en/](https://docs.google.com/presentation/d/16uJIRTJbvknvPyrH_nE-AmLdVBKw8hy6Rk6aAggrpYk/edit?usp=sharing)
+ - [latest](https://docs.google.com/presentation/d/16uJIRTJbvknvPyrH_nE-AmLdVBKw8hy6Rk6aAggrpYk/edit?usp=sharing)(BDX.io Bordeaux, France - 2017 /en/)
```
Comment (delphinemillet) [inline README.md]: rollback to the previous commit...
You win :stuck_out_tongue_closed_eyes:

**Review by delphinemillet — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (delphinemillet) [reply README.md]: ok I missed the commit with the space fixed. Can you capitalize `Latest` ?

**Review by delphinemillet — APPROVED** [review]

**Pull request merged**